ros2_batch_job/osx_batch/__init__.py

```diff
@@ -47,6 +47,9 @@ def pre(self):
               os.environ['OPENSSL_ROOT_DIR'] = brew_openssl_prefix_result.stdout.decode().strip('\n')
         if 'OSPL_HOME' not in os.environ:
             os.environ['OSPL_HOME'] = os.path.join(os.environ['HOME'], 'opensplice', 'HDE', 'x86_64.darwin10_clang')
+        # TODO(wjwwood): remove this when qt5 is linked on macOS by default
+        # See: https://github.com/Homebrew/homebrew-core/issues/8392#issuecomment-334328367
+        os.environ['CMAKE_PREFIX_PATH'] = os.environ.get('CMAKE_PREFIX_PATH', '') + ':/usr/local/opt/qt'
```

Comment:
It's a minor nit as this script is inherently macos specific but above `os.pathsep` is used rather than the literal ':'.

Reply:
No worries, it's an easy fix :+1: